.eslintrc.js

```diff
@@ -71,6 +71,8 @@ module.exports = {
 
 
     // allow debugger during development only
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+
+    "new-cap": 0
```

Comment:
Why did you add this rule instead of importing `shajs` constructor as `Shajs` for instance?

Reply:
Done, I import sha256 as Sha256 now

```{js}
import { sha256 as Sha256 } from 'sha.js'
```